src/components/Breadcrumb/BreadcrumbItem.tsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import { getCommonProps } from '../../utils';
+import classnames from 'classnames';
+import chevronNext from '../../assets/chevronNext.svg';
+
+export interface BreadcrumbItemProps extends React.HTMLAttributes<HTMLDivElement> {
+  /**
+   * Hypertext reference to navigate to breadcrumb item
+   */
+  href?: string;
+  /**
+   * Whether the user is on the current item page or not
+   */
+  isCurrent?: boolean;
+  /**
+   * Label to display in the breadcrumb
+   */
+  label?: string;
+}
+
+const BreadcrumbItem = ({ className, href, label, isCurrent = false, ...props }: BreadcrumbItemProps) => {
+  const { className: baseClassName, ...commonProps } = getCommonProps(props, 'Breadcrumb');
+  const extraClass = !isCurrent ? 'previous' : 'current';
+  const ariaCurrent = !isCurrent ? false : 'page';
+  const currentHref = !isCurrent ? href : '';
+
+  return (
+    <li>
+      <a
```

Comment:
Could we utilize our [link](https://deploy-preview-205--phillips-seldon.netlify.app/?path=/story/components-links-link--playground) component here? this may make styling the hover styles unnecessary.

Reply:
I originally used the Link component but I thought it would be cleaner to use an anchor component because there were several styling overrides I was going to need to do anyway. This component's hover style differs from the link component, I only saw underlined variations. I will look into it again, though.

Reply:
I'm slightly torn here. I do think we should be using the link component here for consistency but the default link component would require style overrides. Maybe this is another case for a LinkVariant like we have in the header and some other places. What do you think @davidicus?

Reply:
Does the list variant work here?

Reply:
Was just looking into this, I missed the link variation first time around. This gets it somewhat close but there would still need to be several overrides to make it consistent with Figma. The font, capitalization, and color all need to be overridden and looks like when I attempt to do so I mess up the hover styling. Maybe I am doing it incorrectly? @davidicus @adietrich3074 

```
.#{$px}-breadcrumb {  
  .#{$px}-text{
    @include text($body3);
  }
  ...
}
```

^ seems to break the hover styling.

Reply:
Let's not fight it if there are too many over rides. I do think we should speak with design about these discrepancies but we can get this in and iterate after the fact.